guides/concepts/index.md

```diff
@@ -1 +1,100 @@
-From an outside perspective it is clear that a lot of the underlying concepts in Cardstack reflect some of the useful architectural decisions of Ember. If you are familiar with Ember then you are likely going to understand the most/all of these concepts without too much effort.
+This section is a crash course in Cardstack concepts. It includes vocabulary that you'll find throughout the other topics, and it draws some parallels to other frameworks and JavaScript in general.
+
+
+## Essential Vocabulary
+
+### Cards
+
+Cards are reusable units of functionality - like a blog article, a text editor, a payment system, and more.
+A developer would use many cards, and cards inside of cards, to create the end user experience.
+Each card can have its own data schema, which flows from the front end to the data layer. When someone is looking at a card, it is reflected in the browser URL.
+The information contained in that URL informs communication with the back end servers.
+
+### The Cardstack mono-repo
+
+The [Cardstack mono-repo](https://github.com/cardstack/cardstack) is a collection of the various elements that could be used to create a Cardstack environment.
+It contains some commonly-used Cards, the Cardstack Hub that connects data sources, and the code that ties everything together behind the scenes.
+
+### Plugins and Data Adapters
+
+A plugin, or data adapter, is made up of four functions needed to connect with a new data source.
+For example, someone could write or use plugins to connect to GitHub, Firebase, and a blockchain source.
+That same plugin code could be used by anyone running a Cardstack environment who wants to connect to the same data source.
+
+### The Hub
+
+The Cardstack Hub is where data from many sources come together.
```

Comment:
I like how @ef4 defines it as being sort of like a smart caching layer that indexes data from the different data sources, and makes it _quickly_ available for the frontend. We can probably also elaborate a bit on the "heavy lifting" that the hub does.

For example, the hub will asynchronously crawl the data sources (which I think we can define the intervals?) and throw them into an index, to make data access a little more quick than fetching at realtime. When the hub indexes data, it already preprocesses some of the JSON that will eventually get returned on an API call

Reply:
added!